### PR TITLE
feat: add employees component

### PR DESCRIPTION
**Pull request by punwai**
## Description
- Add employees component

## Related issues and discussion
#156

## Screenshots, if any
employee

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/organization/employees/employees.component.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      <ng-container matColumnDef="officeName">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Office </th>
+        <td mat-cell *matCellDef="let employees"> {{ employees.officeName }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="isActive">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <td mat-cell *matCellDef="let employees">
+          <div [className]="employees.isActive === true ? 'active' : 'inactive'">
+            <fa-icon matTooltip="{{ employees.isActive === true ? 'Active' : 'Inactive' }}" matTooltipPosition="right" icon="stop" size="lg"></fa-icon>
```
Comment (abhaychawla) [inline src/app/organization/employees/employees.component.html]: Let's update this (icon, colors whatever necessary) as per the components currently implemented in the web app: for example, System => Hooks, etc. to ensure consistency. We are using circular icons here.

File: src/app/organization/employees/employees.component.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      cursor: pointer;
+  }
+}
+
+.active {
+  color: #32cd32;
+}
+
+.inactive {
+  color: #666f73;
```
Comment (abhaychawla) [inline src/app/organization/employees/employees.component.scss]:
```suggestion
  color: #f44366;
```

File: src/app/organization/employees/employees.component.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="displayName">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+        <td mat-cell *matCellDef="let employee"> {{ employee.displayName }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="isLoanOfficer">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Is Loan Officer </th>
```
Comment (abhaychawla) [inline src/app/organization/employees/employees.component.html]:
```suggestion
        <th mat-header-cell *matHeaderCellDef mat-sort-header> Loan Officer </th>
```

File: src/app/organization/employees/employees.component.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        <td mat-cell *matCellDef="let employee"> {{ employee.isLoanOfficer }} </td>
```
Comment (abhaychawla) [inline src/app/organization/employees/employees.component.html]: Let's update this with icons similar to what's currently implemented in the web app components like System => Codes etc.

**Comment by punwai** [comment]
I have added the necessary changes. I have:
- Changed the icons to circular icons
- Changed Loan Officer to circular icons
- Loan Officer Tooltip: Yes, No
- Status Tooltip: Active, Inactive

Screenshot 2019-12-07 at 23 12 54
Screenshot 2019-12-07 at 23 13 08
Screenshot 2019-12-07 at 23 13 11

File: src/app/organization/employees/employees.component.html (hunk 4 above)
Comment (abhaychawla) [reply src/app/organization/employees/employees.component.html]: @punwai For loan officer as I specified take System => Codes as an example where we have icons `check-circle` and `times-circle`.

**Comment by punwai** [comment]
Screenshot 2019-12-08 at 09 18 49

**Comment by punwai** [comment]
Made the necessary changes :)

**Pull request merged**